### DISCUSSION TITLE
[Console] CommandTester with multiline responses to console questions

### DISCUSSION THREAD

**Discussion by misaert** [symfony/symfony#58379] [Q&A]
Hi,

Great work with https://github.com/symfony/symfony/pull/37683 to have the possibility of multiline question. (and https://github.com/symfony/symfony/pull/38351)

I am wondering how to test it with another question on the same command?

On my tests, it is always the content of the first multiline question even if I have two inputs.

Code
```php
$io = new SymfonyStyle($input, $output);
$question = new Question('First question');
$question->setMultiline(true);
$answered1 = $io->askQuestion($question);

$answered2 = (bool) $io->askQuestion(
    new ConfirmationQuestion(
        'Second question',
        default: false
    )
);
```

Test:
```php
$commandTester->setInputs([file_get_contents(__DIR__ . '/multilines.txt'), 'yes']);
```

`yes` is set on the multiline question instead of the second question.

Maybe @ramsey , do you have an idea?

Thanks for the help,

Mickaël Isaert

**Comment by misaert** [comment]
Hi,

Maybe should I create issue from this discussion?

Thanks,

Mickaël Isaert

**Comment by MatTheCat** [comment]
Hello, did #61501 fix your use-case?

**Comment by misaert** [comment]
Hello @MatTheCat ,

Yes, thank you :+1:
